Personel/Adesh/7sep/sumOfOddvalues.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+int 
+main(){
+	int i, num[5], sum=0;
+	printf("\nInput the 1st number: "); 
+    scanf("%d", &num[0]);
+    printf("\nInput the 2nd number: "); 
+    scanf("%d", &num[1]);
+    printf("\nInput the 3rd number: "); 
+    scanf("%d", &num[2]);
+	printf("\nInput the 4th number: "); 
+    scanf("%d", &num[3]);
+    printf("\nInput the 5th number: "); 
+    scanf("%d", &num[4]);
```

Comment:
take the number of input value from user and  use for loop for taking user inputs